src/passes/StringLowering.cpp

```diff
@@ -23,7 +23,10 @@
 //
 // StringLowering does the same, and also replaces those new globals with
 // imported globals of type externref, for use with the string imports proposal.
-// String operations will likewise need to be lowered. TODO
+//
+// A pass argument allows customizing the module name for string constants:
+//
+//   --pass-arg=string-lowering-const-module@MODULE_NAME
```

Comment:
Could we maybe have just a single flag like `--string-constants-module=MODULE_NAME` and as an implementation detail set the pass arguments for both `string-lowering` and `string-lifting` at the same time?

Reply:
It would still need to be a pass arg, so `--pass-arg=string-constants-module@MODULE_NAME`, unless you mean adding it to `wasm-opt.cpp`?

Reply:
Yes, I meant adding it to `wasm-opt.cpp` or the tool-options file.

Reply:
We've intentionally done this in a modular way so far, leaving pass options for passes themselves.

Your suggested user-facing flag would be nicer in a way, I agree, but it is also nice that users know that pass flags have a consistent naming (making it easy to find the flags for a pass).

Reply:
Since the same argument needs to be passed to two different passes in this case, I'm not sure the benefit of modularity outweighs the drawback of having to verbosely duplicate the argument.

How does one find the flags for a pass? It doesn't look like they are documented anywhere rather than in the source. E.g. the help tests are not modified here.

Reply:
Yeah, the source atm is how we document. We could add a `addToHelpText()` method perhaps.

But, what about `--pass-arg=string-constants-module@MODULE_NAME`? We can use that same pass argument in both passes. That is a break from convention in that passes would share an arg, but it is the same arg, so I would be ok with not using the pass name as the prefix. That would avoid the duplication, and give all the benefits, aside from having the `--pass-arg` prefix. What do you think?

Reply:
I see, that sounds fine. We should definitely surface these arguments in the help text somehow. I don't expect users to be reading our source.

Reply:
Ok, renamed to that shared pass-arg flag.